shuup_tests/admin/test_media_module.py

```diff
@@ -8,8 +8,11 @@
 import json
 import tempfile
 
+import mock
```

Comment:
We could probably just be using `unittest.mock`, but since this was used everywhere else as well.